tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -12,11 +12,13 @@
 # Summary: FIPS mozilla-nss test for firefox
 # Maintainer: mitiao <mitiao@gmail.com>,
 #             wnereiz <wnereiz@fsf.member.org>
+# Tag: poo#47018
```

Comment:
I think the "poo#47018" in "Tag" is not the original poo when this test case was added.
And, add another Tag tc#$testcase_number is better if it exists.

Reply:
Actually, the tc# is not that mandatory for the openQA test case, because the case is slightly different from our original tc#. Furthermore, the test case number will be outdated in the next product SPx release. Thanks.

Reply:
OK! thanks for the explanation.